lib/wallet/http.js

```diff
@@ -439,6 +439,7 @@ class HTTP extends Server {
         subtractFee: valid.bool('subtractFee'),
         subtractIndex: valid.i32('subtractIndex'),
         depth: valid.u32(['confirmations', 'depth']),
+        useSelectEstimate: valid.bool('useSelectEstimate'),
```

Comment:
look at `bwallet-cli` and add necessary new option, in help as well!

Reply:
The addition of option in CLI and RPC will be a separate PR. I'll add a new option in help for now.